### PR TITLE
Add Library Page and loader for that page

### PR DESCRIPTION
**Pull request by ivannissimrch**

### CODE REVIEW

File: src/helpers/loaders/getPlayList.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import SpotifyWebApi from "spotify-web-api-js";
+
+export default async function getPlayList(
```
Comment (xMartin) [inline src/helpers/loaders/getPlayList.ts:3]: Should be called "`getPlayLists`"

File: src/helpers/loaders/getPlayListSongs.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default async function getPlayListSongs({ playlistId }, spotifyApi) {
```
Comment (xMartin) [inline src/helpers/loaders/getPlayListSongs.ts:1]: Arguments are missing types.

File: src/helpers/loaders/getPlayListSongs.ts
Comment (xMartin) [inline src/helpers/loaders/getPlayListSongs.ts:1]: This file seems to be unused.

File: src/pages/Library.tsx [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+import Typography from "@mui/material/Typography";
+import { CardActionArea } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import HeadPhonesImage from "../assets/images/headphones.jpg";
 
 export default function LibraryPage() {
+  const lists = useLoaderData() as SpotifyApi.PlaylistObjectSimplified[];
+  const navigate = useNavigate();
+
+  function handleOnClick(id: string) {
```
Comment (xMartin) [inline src/pages/Library.tsx:15]: The `on` in the name is not necessary. The function handles something. In this case a click on an item or playlist.

I'd call it `handleItemClick` or `handlePlaylistClick`.

**Review by xMartin — COMMENTED** [review]
👍

File: src/helpers/loaders/getPlayListSongs.ts
Comment (ivannissimrch) [reply src/helpers/loaders/getPlayListSongs.ts:1]: I accidentally included this function in this pull request. I don't use it here, but I will use it as a loader for the playlist page on pull request #10 branch PlayListSongs.

File: src/pages/Library.tsx (hunk 3 above)
Comment (ivannissimrch) [reply src/pages/Library.tsx:15]: thanks

**Pull request closed without merging**